<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductManagerWindow</class>
 <widget class="QDialog" name="ProductManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Products</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="filterGroup">
     <property name="title">
      <string>Filters &amp; Search</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_name">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filter_name">
        <property name="placeholderText">
         <string>Model name...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_type">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filter_type">
        <item>
         <property name="text">
          <string>All Types</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Commercial</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fighter</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_speed">
        <property name="text">
         <string>Min Speed:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="filter_min_speed">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QTableWidget" name="table_products">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Model</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Speed (km/h)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price ($)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sold Units</string>
      </property>
     </column>
    </widget>
   </item>

   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>Add Product</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="text">
        <string>Delete Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
</ui>
